packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx

```diff
@@ -115,14 +116,22 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps): Re
               </CustomScrollbar>
               <div className={styles.rightSide}>
                 <div className={styles.title}>
-                  <TimePickerTitle>Specify time range</TimePickerTitle>
-                  <div className={styles.description}>
-                    Specify a relative time range, for more information see{' '}
-                    <a href="https://grafana.com/docs/grafana/latest/dashboards/time-range-controls/">
-                      docs <Icon name="external-link-alt" />
-                    </a>
-                    .
-                  </div>
+                  <TimePickerTitle>
+                    <Tooltip
+                      content={
+                        <>
+                          <div>Supported formats</div>
+                          <code className={styles.tooltip}>now-[digit]s/m/h/d/w</code>
```

Comment:
As we have a tooltip now, maybe we could add a few examples of the time ranges?

Reply:
I think this is a good idea for users that might not be very familiar with the syntax